### ISSUE TITLE
can i get some help regarding your post [How to train a new language model from scratch..]?

### ISSUE THREAD

**Issue by do9380a1** [huggingface/blog#115]
I followed up the code one by one and it was great help!
Btw, when I ran the code below, I got 52000 of embedding size ([1,12,52000]). But since 'max_postition_embeddings=514' in the Config, I think embedding size should be 512, not 52000 (which is the vocab_size). 
----------
>inputs = tokenizer("Mi estas Julien.", return_tensors="pt")
>outputs = model(**inputs)
>outputs.logits.shape
 => torch.Size([1,12,52000])
---------
I am quite a beginner in this  area, please correct me if I misunderstood.
For your reference, I tries to pull out the embedding vector of each token to do cosine similarity.
Thanks.
